src/install/haproxy/intro.mdx

```diff
@@ -0,0 +1,36 @@
+---
+headingText: Before we start
+componentType: default
+---
+
+Our HAProxy [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) collects and sends inventory and metrics from your HAProxy instance to the New Relic platform, where you can aggregate and visualize key performance metrics for the HAProxy server, frontends, and backends.   
```

Comment:
Remove this link? Feels random and I'm not sure you've used this pattern in the other docs